app/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+from app.routes.incident_routes import route_url_create, route_url_get_all,route_url_delete_one
```

Comment:
@Wabuluka 
The tests in the test_user.py file will fail because you have not imported those routes properly

Reply:
how am I supposed to import them

Reply:
<img width="952" alt="screenshot 2019-01-16 at 22 02 00" src="https://user-images.githubusercontent.com/21138053/51272150-6172f700-19da-11e9-8b0a-86ab208e12cc.png">